packages/nimble-components/src/anchor-menu-item/index.ts

```diff
@@ -35,7 +35,7 @@ export class AnchorMenuItem extends AnchorBase {
      * @internal
      */
     @observable
-    public startColumnCount: MenuItemColumnCount = 0;
+    public startColumnCount?: MenuItemColumnCount;
```

Comment:
Minor, but seems more appropriate not to have an `"indent-0"` class set on the item initially (though the normal `nimble-menu-item` initially has the class `"indent-undefined"` because FAST template doesn't handle `undefined` properly).

Reply:
`startColumnCount` is assigned to menu items only by the menu, except for the initialization in `AnchorMenuItem` that I removed. I felt it would be more expected that the item not have a `startColumnCount` (and not have any `"indent-N"` class set on it) except when the menu assigns it. However, the regular `MenuItem` already (weirdly) applies an `"indent-undefined"` class before acted on by the menu, so that kind of undercuts my intent. I'm fine reverting back to initializing it to 0.